model-server/build.gradle.kts

```diff
@@ -107,7 +107,7 @@ val cucumber = task("cucumber") {
     }
 }
 
-tasks.named("test") {
+tasks.named("build") {
```

Comment:
Would it make sense to migrate this setup to [test suites](https://blog.gradle.org/introducing-test-suites)?

Reply:
Maybe in a separate PR.